docs/DeveloperGuide.md

```diff
@@ -1,13 +1,39 @@
 # Developer Guide
```

Comment:
Might be good to include a short introduction to the project as it took me awhile to understand the application's target user